sdgym/synthesizers/realtabformer.py

```diff
@@ -0,0 +1,24 @@
+"""REaLTabFormer integration."""
+
+from sdgym.synthesizers.base import BaselineSynthesizer
+
+
+class RealTabFormerSynthesizer(BaselineSynthesizer):
+    """Custom wrapper for the REaLTabFormer synthesizer to make it work with SDGym."""
+
+    def _get_trained_synthesizer(self, data, metadata):
+        try:
+            from realtabformer import REaLTabFormer
+        except Exception as exception:
+            raise ValueError(
+                "In order to use 'RealTabFormerSynthesizer' you have to install sdgym as "
+                "sdgym['realtabformer']."
+            ) from exception
+
+        model = REaLTabFormer(model_type='tabular')
+        model.fit(data, device='cpu')
```

Comment:
This spits a lot of output while running the benchmark, could we silence it ? For example:
<img width="665" alt="image" src="https://github.com/user-attachments/assets/0e3eabc5-a36a-4308-b7cc-160ec77d3c16">

Reply:
You have to use this decorator:

```python
@contextlib.contextmanager
def prevent_tqdm_output():
    """Temporarily disables tqdm m."""
    tqdm.__init__ = partialmethod(tqdm.__init__, disable=True)
    try:
        yield
    finally:
        tqdm.__init__ = partialmethod(tqdm.__init__, disable=False)
```